kolibri/plugins/management/assets/src/vue/manage-content-page/wizard-import-local.vue

```diff
@@ -12,32 +12,44 @@
     <div class="main">
       <template v-if="!drivesLoading">
         <div class="modal-message">
-          <h2 class="core-text-alert" v-if="drivesWithData.length === 0">
+          <h2 class="core-text-alert" v-if="!wizardState.driveList">
```

Comment:
If `wizardState.driveList` can be `undefined` or `null`, presumably that means we have some pretty funky values (or even errors) coming from our computed properties - because we can't call filter from `undefined` or `null`, could you clean up the computed properties below as well to make sure they don't throw errors?